app/Events/PhoneNumberQueried.php

```diff
@@ -17,12 +17,18 @@ public static function webhook(Request $request, SmsCommand $command): string
     {
         // TODO: Account for numbers with spaces :/
```

Comment:
the `phone_number` helper handles this

Reply:
@hans-lts can you make a code suggestion? I just moved existing code.

Reply:
😬 I may have been a little too hasty with that comment... 

My tests for the number with spaces fail and I'm a little too tired to get it handled right now. I will re-approach in the morning.